tests/feature_map/test_layered_feature_map.py

```diff
@@ -194,11 +213,37 @@ def add_AA(gate_function, p, offset=0):
         for lfm_gate, qiskit_gate in zip(lfm_list, qiskit_list):
             lfm_gate("p", ent_strategy="NN", encoding=np.arccos)
             offset = add_NN(qiskit_gate, p, offset)
-            assert str(lfm.get_circuit([], p)) == str(expected_circuit)
+            assert str(lfm.get_circuit(x, p)) == str(expected_circuit)
 
             lfm_gate("p", ent_strategy="AA", encoding=np.arccos)
             offset = add_AA(qiskit_gate, p, offset)
-            assert str(lfm.get_circuit([], p)) == str(expected_circuit)
+            assert str(lfm.get_circuit(x, p)) == str(expected_circuit)
+
+        kernel = FidelityKernel(
+            lfm, Executor("statevector_simulator"), initial_parameters=0.5 * np.ones(72)
+        ).evaluate(np.array([[]]), np.array([[]]))
+        assert np.allclose(kernel, np.array([1.0]))
+
+        # Won't run because of a bug in qiskit
+        # See https://github.com/Qiskit/qiskit/issues/8723
```

Comment:
Then remove it? 😄

Reply:
Well it is implemented an should work once the qiskit bug is fixed. I would like to keep it.

Reply:
Alright